builtin/game/chatcommands.lua

```diff
@@ -487,6 +487,26 @@ core.register_chatcommand("deleteblocks", {
 	end,
 })
 
+core.register_chatcommand("fixlight", {
+	params = "(here [radius]) | (<pos1> <pos2>)",
+	description = "Removes lighting bugs in the area between pos1 and pos2 "
+		.. "(light might actually change in a larger volume)",
```

Comment:
I don't think this line is needed, it isn't present in other similar commands, the warning can be in documentation instead.